packages/calypso-stripe/src/index.tsx

```diff
@@ -272,6 +287,27 @@ export async function confirmStripePaymentIntent(
 	return paymentIntent;
 }
 
+/**
+ * Return Object with mandate option values to be included in creating a SetupIntent
+ *
+ * @param {ShoppingCart} ShoppingCart The shopping cart for the SetupIntent
+ * @returns {Object} An object keyed by the fields needed for the mandate options
+ */
+function getMandateOptionsFromShoppingCart(
```

Comment:
I wonder if it would be better to add this function to the shopping cart component and let it be exported from there? Rather than trying to import the cart here and then finagle any validation we would need.